azure-pipelines/ui-automation/templates/build-msal-automation-app.yml

```diff
@@ -23,6 +23,7 @@ parameters:
   - name: msalVersion
     displayName: MSAL Version
     type: string
+    default: "*"
```

Comment:
probably put `0.0.+` 

that would get the latest

Reply:
Gotcha, I wasn't sure what the correct default value should be here.